src/main/java/com/revature/domain/PersonBatchJoin.java

```diff
@@ -0,0 +1,103 @@
+package com.revature.domain;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+
+@Entity
+@Table(name="ie_person_batch")
+public class PersonBatchJoin {
+
+	@Id
+	@Column(name="join_id")
+	@GeneratedValue(strategy=GenerationType.SEQUENCE, generator="perBatSeq")
+	@SequenceGenerator(allocationSize=1, name="perBatSeq", sequenceName="person_batch_seq")
+	private int id;
+	
+	@ManyToOne(fetch = FetchType.EAGER)
+	@JoinColumn(name = "person_id", nullable = false)
+	private Person person;
+	
+	@ManyToOne(fetch = FetchType.EAGER)
+	@JoinColumn(name = "batch_id", nullable = false)
+	private Batch batch;
+
+	public int getId() {
+		return id;
+	}
+
+	public void setId(int id) {
+		this.id = id;
+	}
+
+	public Person getPerson() {
+		return person;
+	}
+
+	public void setPerson(Person person) {
+		this.person = person;
+	}
+
+	public Batch getBatch() {
+		return batch;
+	}
+
+	public void setBatch(Batch batch) {
+		this.batch = batch;
+	}
+
+	public PersonBatchJoin(){}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1186)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this constructor to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1213)